src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Driver
 {
+    [Obsolete("AnalyzeCommandBase will be deprecated entirely soon. Use MultithreadedAnalyzeCommandBase instead.")]
```

Comment:
>AnalyzeCommandBase [](http://example.com/codeflow?start=15&length=18)

It grows wearisome to maintain this obsolete single-threaded analysis, let's zap it next release.